frontend/src/scenes/trends/ActionsLineGraph.js

```diff
@@ -4,9 +4,10 @@ import { LineGraph } from './LineGraph'
 import { useActions, useValues } from 'kea'
 import { trendsLogic } from 'scenes/trends/trendsLogic'
 
-export function ActionsLineGraph({ dashboardItemId = null, color = 'white', filters: filtersParam }) {
-    const { filters, results, resultsLoading } = useValues(trendsLogic({ dashboardItemId, filters: filtersParam }))
-    const { loadResults, loadPeople } = useActions(trendsLogic({ dashboardItemId, filters: filtersParam }))
+export function ActionsLineGraph({ dashboardItemId = null, color = 'white', filters: filtersParam, cachedResults }) {
+    let logic = trendsLogic({ dashboardItemId, filters: filtersParam, cachedResults })
```

Comment:
nit, but best to have this as a `const` since we will not reassign its value later on